policyengine_us/parameters/gov/states/hi/tax/income/alternative_tax/availability.yaml

```diff
@@ -0,0 +1,11 @@
+description: Hawaii allows for an alternative tax on capital gains calculation if this is true.
+values:
+  2021-01-01: true  
+  2023-01-01: false
+metadata:
+  unit: bool
+  period: year
+  label: Hawaii alternative tax on capital gain availability
+  reference:
+    - title: House of Representatives Thirty-Second Legislature, 2023. State of Hawaii, A Bill For An Act
```

Comment:
could we also add the source that shows its availability in 2022? Legal code + tax form